test/run-tests.py

```diff
@@ -0,0 +1,115 @@
+import contextlib
+import logging
+import os
+import subprocess
+import sys
+import tempfile
+import time
+from enum import Enum
+
+EXPECTED_TIME_TO_BOOT = 60  # seconds
+RESET = "\033[0m"
+BOLD = "\033[1m"
+RED = "\033[31m"
+OBJECTS = os.environ.get("OBJECTS", tempfile.mkdtemp())
+OUTPUT_DIR = os.environ.get("OUTPUT_DIR", tempfile.mkdtemp())
```

Comment:
Please pass `prefix="osbuild-"` or similar, so that I can see which things in `/tmp` I can delete. These can get quite large and I suspect I'll be running this test often.

We'll need to make the tests clean up after themselves at some point, but I'm fine landing it like this for now.